atomate/vasp/firetasks/lobster_tasks.py

```diff
@@ -0,0 +1,349 @@
+# coding: utf-8
+
+"""
+This module defines tasks  that can be used to handle Lobster calculations that are based on VASP wavefunctions.
+"""
+
+import json
+import logging
+import os
+import shutil
+import warnings
+
+from atomate.common.firetasks.glue_tasks import get_calc_loc
+from atomate.utils.utils import env_chk, get_meta_from_structure
+from atomate.vasp.database import VaspCalcDb, put_file_in_gridfs
+from custodian import Custodian
+from custodian.lobster.handlers import ChargeSpillingValidator, EnoughBandsValidator, LobsterFilesValidator
+from custodian.lobster.jobs import LobsterJob
+from fireworks import FiretaskBase, explicit_serialize, FWAction
+from fireworks.utilities.fw_serializers import DATETIME_HANDLER
+from monty.json import jsanitize
+from monty.os.path import zpath
+from monty.serialization import loadfn
+from pymatgen.core.structure import Structure
+from pymatgen.io.lobster import Lobsterout, Lobsterin
+
+__author__ = "Janine George, Guido Petretto"
+__email__ = 'janine.george@uclouvain.be, guido.petretto@uclouvain.be'
+
+LOBSTERINPUT_FILES = ["lobsterin"]
+LOBSTEROUTPUT_FILES = ["lobsterout", "CHARGE.lobster", "COHPCAR.lobster", "COOPCAR.lobster", "DOSCAR.lobster",
+                       "GROSSPOP.lobster", "ICOHPLIST.lobster", "ICOOPLIST.lobster", "lobster.out",
+                       "projectionData.lobster"]
+
+VASP_OUTPUT_FILES = ["OUTCAR", "vasprun.xml", "CHG", "CHGCAR", "CONTCAR", "INCAR", "KPOINTS", "POSCAR", "POTCAR",
+                     "DOSCAR", "EIGENVAL", "IBZKPT", "OSZICAR", "PCDAT", "PROCAR", "REPORT", "WAVECAR", "XDATCAR"]
+VASP_OUTPUT_FILES_without_WAVECAR = ["OUTCAR", "vasprun.xml", "CHG", "CHGCAR", "CONTCAR", "INCAR", "KPOINTS", "POSCAR",

```

Comment:
You have a few of these VASP file lists. Can you just have one instance of `VASP_OUPUT_FILES`, and then if you need a version without a specific file, just dynamically create it when you need it in the code. I.e.

```python
files_to_check = [f for f in VASP_OUTPUT_FILES if f not in ["WAVECAR"]]
```

You should import list of VASP_OUTPUT files in the Lobster module and anywhere else it is needed, rather than redefining it.

Currently, if VASP output files change, you will need to update multiple different lists to keep them up to date.

Reply:
Will change this + will do some cleanup (will only copy the files that are required for a lobsterrun).